### YamlMime:UniversalReference
items:
  - uid: azure-arm-datalake-store.AdlsFileNotFoundException
    name: AdlsFileNotFoundException
    fullName: AdlsFileNotFoundException
    children:
      - azure-arm-datalake-store.AdlsFileNotFoundException.exception
      - azure-arm-datalake-store.AdlsFileNotFoundException.javaClassName
      - azure-arm-datalake-store.AdlsFileNotFoundException.message
    langs:
      - typeScript
    type: interface
    summary: ファイルまたはフォルダーを示すスロー WebHDFS 例外は見つかりませんでした。 404 エラー応答コードが返されます (見つかりません) 場合にスローされます。
    extends:
      name: azure-arm-datalake-store.AdlsRemoteException
    package: azure-arm-datalake-store
  - uid: azure-arm-datalake-store.AdlsFileNotFoundException.exception
    name: exception
    fullName: exception
    children: []
    langs:
      - typeScript
    type: property
    summary: ポリモーフィックな識別子
    syntax:
      content: 'exception: string'
      return:
        type:
          - string
    package: azure-arm-datalake-store
  - uid: azure-arm-datalake-store.AdlsFileNotFoundException.javaClassName
    name: javaClassName
    fullName: javaClassName
    children: []
    langs:
      - typeScript
    type: property
    summary: '''java.lang.IllegalArgumentException'' など、スローされる例外のクラスの完全なパッケージ名。'
    optional: true
    syntax:
      content: 'javaClassName?: string'
      return:
        type:
          - string
    package: azure-arm-datalake-store
  - uid: azure-arm-datalake-store.AdlsFileNotFoundException.message
    name: message
    fullName: message
    children: []
    langs:
      - typeScript
    type: property
    summary: 'など、スローされた例外に関連付けられたメッセージ '' webhdfs パラメーター「権限」に無効な値:...''。'
    optional: true
    syntax:
      content: 'message?: string'
      return:
        type:
          - string
    package: azure-arm-datalake-store
references:
  - uid: azure-arm-datalake-store.AdlsRemoteException
    name: AdlsRemoteException
    spec.typeScript:
      - name: AdlsRemoteException
        fullName: AdlsRemoteException
        uid: azure-arm-datalake-store.AdlsRemoteException